### PR TITLE
Feature/jcs testing

### LINKED ISSUE #54: Location Testing
- [ ] Integrate testing into Location Prop
- [ ] needs mostly render tests 
- [ ] App component has locations fetch call

### LINKED ISSUE #77: Write Tests for Weekly Forecast & Location
`Location`:
-[] Should render city / region data
-[] Should render current weather 
-[] Should render current AQI info
-[] Address imperial part in fetch url to convert visib, wind speed, temp
-[] mock out fetching
-[] test to see if child component `weekly forecast` renders in parent comp `Location`

- precip in current is millimeters in last hour. do we want that?

`Weekly Forecast`:
-[] Should render 7 days with: min and max temp, wind speed and deg, uvi, icon, humidity, cloud cover

### PR DESCRIPTION
**Pull request by JoshSevy**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Testing Testing Testing
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
`Your message here`
# Message for reviewer:
All tests pass looks good
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/App/App.test.tsx [hunk 1]
```diff
@@ -136,22 +143,28 @@ describe('App', () => {
...
+	
+		const title1 = await screen.findByText(/denver/i);
+		
 		fireEvent.click(title1)
-
-		const happeningNowTitle = await findByRole('heading', { name: /happening now/i })
-		const airQualityTitle = await findByRole('heading', { name: /happening now/i })
-		const weeklyForecastTitle = await findByRole('heading', { name: /happening now/i })
+		
+
```
Comment (leighlars) [inline src/App/App.test.tsx:156]: great work!

File: src/Location/Location.test.tsx [hunk 2]
```diff
@@ -1,175 +1,184 @@
...
+        sunrise: 0,
+        sunset: 0,
+        temp: {
+          day: 285,
+          min: 222,
+          max: 285,
+          night: 285,
+          eve: 285,
+          morn: 285,
+        },
```
Comment (leighlars) [inline src/Location/Location.test.tsx:48]: thank you for cleaning this up!

**Review by leighlars — APPROVED** [review]
Thank you for taking on Location and all these tests!

File: src/App/App.test.tsx [hunk 3]
```diff
@@ -78,11 +78,14 @@ describe('App', () => {
 		}
 	})
 
+
+	//keeps throwing a not using act(()=>{}) error i tried wrapping render in act but error still persists
```
Comment (ErinUntermeyer) [inline src/App/App.test.tsx:82]: Definitely need to do a bit more research on the act error and how to solve it.

File: src/Location/Location.test.tsx (hunk 2 above)
Comment (ErinUntermeyer) [reply src/Location/Location.test.tsx:48]: Yes, thank you!

**Comment by ErinUntermeyer** [comment]
Awesome job Josh!

**Review by ErinUntermeyer — APPROVED** [review]

**Pull request merged**